usergroups.go

```diff
@@ -121,16 +122,44 @@ func (api *Client) EnableUserGroupContext(ctx context.Context, userGroup string)
 	return response.UserGroup, nil
 }
 
+// GetUserGroupsParams contains arguments for GetUserGroups call
+type GetUserGroupsParams struct {
+	// IncludeCount	- Include the number of users in each User Group. Default: false
+	IncludeCount bool
+	// IncludeDisabled - Include disabled User Groups. Default: false
+	IncludeDisabled bool
+	// IncludeUsers - Include the list of users for each User Group. Default: false
+	IncludeUsers bool
+}
+
+// NewGetUserGroupsParams creates a new instance GetUserGroupsParams with defaults
+func NewGetUserGroupsParams() GetUserGroupsParams {
+	return GetUserGroupsParams{
+		IncludeCount:    false,
+		IncludeDisabled: false,
+		IncludeUsers:    false,
+	}
+}
+
 // GetUserGroups returns a list of user groups for the team
-func (api *Client) GetUserGroups() ([]UserGroup, error) {
-	return api.GetUserGroupsContext(context.Background())
+func (api *Client) GetUserGroups(params GetUserGroupsParams) ([]UserGroup, error) {
```

Comment:
as this breaks the API, I would recommend switching to an (options ...option) pattern. plenty of examples in the library.